specs/HomeScreen.spec.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import * as Redux from 'react-redux'
+import { render } from '@testing-library/react-native'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import HomeScreen from '../src/components/HomeScreen'
+import initialState from '../src/state/store/initialState'
+
+const posts = [
```

Comment:
You couldn't import the fixture file here? :)

Reply:
we tried this at first, but since the test relies on mocking quite a bit we ended up in a sort of jest-props-hell bottlenecked the feature, so we've left it as is- this will also help as a framework for future tests to help us follow jest's process.